### PR TITLE
Fix typos and remove unused imports

### PR DESCRIPTION
**Pull request by bsipocz**
Trivial fixes in coordinates, table and nddata

### CODE REVIEW

File: astropy/table/table.py [hunk 1]
```diff
@@ -630,7 +628,7 @@ def show_in_browser(self,
             the table in memory).  A negative value of ``max_lines`` indicates
             no row limit
         jsviewer : bool
-            If `True`, prepends some javascript headers so that the table is
+            If `True`, pretends some javascript headers so that the table is
```
Comment (astrofrog) [inline astropy/table/table.py]: Here, ``prepends`` was correct (opposite of appends - see http://en.wiktionary.org/wiki/prepend)

**Comment by astrofrog** [comment]
@bsipocz - thanks! Just one comment above. How do you find these, do you have a Python-friendly spell-checker?

File: astropy/table/table.py (hunk 1 above)
Comment (bsipocz) [inline astropy/table/table.py]: Sorry about that, I should have known that word.

Comment (astrofrog) [inline astropy/table/table.py]: No problem!

**Comment by bsipocz** [comment]
@astrofrog - Sadly I haven't found any useful tool for this, and just run the spell checker in emacs.

**Comment by astrofrog** [comment]
Thanks!

**Pull request merged**